lib/OpenQA/Shared/Controller/Running.pm

```diff
@@ -113,8 +113,8 @@ sub streamtext ($self, $file_name, $start_hook = undef, $close_hook = undef) {
     # Setup utility function to close the connection if something goes wrong
     my $timer_id;
     my $close_connection = sub ($self) {
+        log_debug "Connection to $worker will be terminated because $logfile changed";
```

Comment:
This looks wrong as `$worker` is not containing a merge ID/name. I also doubt that this log message is generally very useful during operation so either reduce the level to `trace` or remove it.

Reply:
I suggested this log message above in https://github.com/os-autoinst/openQA/pull/6649#issuecomment-3174414209
> I think the fix could include at least a debug message that reports that the file was shrunk, or the inode changed. This shouldn't happen often, but if something is going wrong, we would have that in the logs.

So maybe you could comment on that. If we give @d3flex different suggestions, what should he do?